Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -118,9 +118,29 @@
     color : "color() * acos(degrees(${temperature})) + color() * asin(${temperature}) + color() * atan(${temperature})"
 });
 
+addStyle('ATan2', {
+    color : "color() * atan2(${POSITION}[0],${temperature}) + color() * atan2(${POSITION}[1],${temperature}) + color() * atan2(${POSITION}[2],${temperature})",
+    pointSize : "5"
+});
+
 addStyle('Sqrt', {
     color : "color() * sqrt(${temperature})",
-    pointSize : "10"
+    pointSize : "5"
+});
+
+addStyle('Pow', {
+    color : "color() * pow(${temperature}, 3)",
+    pointSize : "5"
+});
+
+addStyle('Min', {
+    color : "color() * min(${POSITION}[0],0.75)",
+    pointSize : "5"
+});
+
+addStyle('Max', {
+    color : "color() * max(${POSITION}[2],0.25)",
+    pointSize : "5"
```

Comment:
Min and max could be combined into one style, min could affect the red channel and max could affect the green channel.